windowsdoc.md

```diff
@@ -0,0 +1,75 @@
+## **Windows Compatible Cwltool**  
+Windows Compatible cwltool means that now you can create and run your workflows using cwltool on Windows under a [Docker Container](https://docs.docker.com/docker-for-windows/).
+On Windows, all tools and workflows will be executed inside a linux based Docker Container. If a Docker Container is not provided explicitly (using [Docker Requirement](http://www.commonwl.org/v1.0/CommandLineTool.html#DockerRequirement)) by user, we use a minimal, posix compliant Alpine Docker image which has Bash pre installed as fallback (Default) [Docker Container](https://github.com/frol/docker-alpine-bash).  
```

Comment:
-> 
If a Docker Container is not explicitly provided (using a [Docker Requirement](http://www.commonwl.org/v1.0/CommandLineTool.html#DockerRequirement)) by the user, then a minimal, posix compliant Alpine Docker image which has Bash pre installed is used as a fallback (Default) [Docker Container](https://github.com/frol/docker-alpine-bash).